src/main/kotlin/org/cascadebot/cascadebot/utils/SerializableMessage.kt

```diff
@@ -0,0 +1,93 @@
+package org.cascadebot.cascadebot.utils
+
+import club.minnced.discord.webhook.send.WebhookEmbed
+import club.minnced.discord.webhook.send.WebhookEmbedBuilder
+import net.dv8tion.jda.api.entities.Message
+import net.dv8tion.jda.api.entities.Message.MessageFlag
+import net.dv8tion.jda.api.entities.MessageEmbed
+import net.dv8tion.jda.api.entities.MessageType
+import java.time.OffsetDateTime
+import java.util.stream.Collectors
+
+class SerializableMessage(
```

Comment:
We should look into if we can use JDAs serialization stuffs so this class doesn't need to exist